packages/app-builder-lib/src/configuration.ts

```diff
@@ -141,6 +141,11 @@ export interface Configuration extends PlatformSpecificBuildOptions {
    */
   readonly buildVersion?: string | null
 
+  /**
+   * Whether to download the alternate FFmpeg library from Electron's release assets and replace the default FFmpeg library prior to signing
+   */
+  readonly downloadAlternateFFmpeg?: boolean
```

Comment:
What are your thoughts on renaming this to: `injectElectronFFmpeg` or something similar? `downloadAlternate` seems odd to me because it can also use `ELECTRON_BUILDER_CACHE` and this seems to be more Electron-specific than just a generic alternate library.

Reply:
yeah, a name for this is kind of hard. It's also not just `downloading` it is really `using` the alternate ffmpeg library. I was also hesitant to call it `nonProprietary` since I don't really have a guarantee from electron that it is non proprietary...we as consumers are supposed to be the ones checking this sort of thing. Granted, it's built with the `proprietary_codecs=false` flag from chromium.

Anyway, all Electron's come with this ffmpeg library, its just that we want to _use_ an alternative one that supposedly has only non proprietary codecs in it. With that, I'd say the best name maybe is `useNonProprietaryFFmpeg` ... that seems to be as descriptive and accurate as it gets.